test/lib/cql_test_env.cc

```diff
@@ -109,6 +109,7 @@ cql_test_config::cql_test_config(shared_ptr<db::config> cfg)
 
     db_config->add_cdc_extension();
     db_config->add_per_partition_rate_limit_extension();
+    db_config->add_tags_extension();
```

Comment:
Why can't we add all extensions here, and call the same add_all_extensions() from main?

Preexisting problem, so not a blocker.